scannerc.go

```diff
@@ -857,6 +857,7 @@ func yaml_parser_fetch_next_token(parser *yaml_parser_t) (ok bool) {
 	//switch parser.buffer[parser.buffer_pos] {
 	//case '-', '?', ':', ',', '?', '-', ',', ':', ']', '[', '}', '{', '&', '#', '!', '*', '>', '|', '"', '\'', '@', '%', '-', '`':
 	//}
+	//nolint:staticcheck // De Morgan's law really doesn't need to be applied here.
```

Comment:
Considering you are adding nolint at a few place, please add nolintlint golangci-lint linter and configure it to require-explanations